py-polars/polars/testing/asserts.py

```diff
@@ -570,3 +570,10 @@ def assert_frame_equal_local_categoricals(df_a: DataFrame, df_b: DataFrame) -> N
     assert_frame_equal(df_a.with_columns(cat_to_str), df_b.with_columns(cat_to_str))
     cat_to_phys = F.col(Categorical).to_physical()
     assert_frame_equal(df_a.with_columns(cat_to_phys), df_b.with_columns(cat_to_phys))
+
+
+def assert_repr_equals(item: Any, expected: str) -> None:
```

Comment:
I don't think we want this in the public API, at least for now. You can define it in the test module you're using it in or maybe in `conftest.py`. I did something similar for `assert_expr_equal` somewhere.

Reply:
Sure; I'll move somewhere else (it's used in more than one test file, but it doesn't need to be part of our own public API, quite right)